Makefile

```diff
@@ -60,17 +60,17 @@ build:
 # because pip install doesn't handle
 # INSTALL_REQUIRES properly
 #: Set up to run from the source tree
-develop:  mathics/data/op-tables.json
+develop:
```

Comment:
The removal of the dependency in here I don't understand. Why should this be removed?

Reply:
It's no longer needed because `setup.py` now builds this file by itself

Reply:
Using (most? or all?) of the Makefile is optional. 

Its purpose for old-timers like me is to allow various common tasks to be done in the same way as other projects that have Makefiles.  So instead of remembering the common lingo for running tests (which has changed over time) , I can run "make check" which insteads changes from say "nosetest" to "pytest" (to whatever it is going to be next year and or next decade). 

But specifically, that dependency is there to see if the optables needs to be created. Without this, whenever someone runs "make develop" the optable will get created, whether or not it was there.

Reply:
If you don't want `setup.py` to unconditionally generate `op-tables.json`, we can put in a check there; but in any case, there is no longer a need for the Makefile rule for building `mathics/data/op-tables.json`

Reply:
> If you don't want `setup.py` to unconditionally generate `op-tables.json`, we can put in a check there

Done in cb38d92b

Reply:
We are talking at cross purposes. The change  make are mechanisms like adding Python code, or whether to use this library call, or some other and so on.

The purpose of the Makefile is to present, at least to me and other people who work across many projects on many platforms using different languages, a common _interface_ or way to do common kinds of things in projects. 

If a file is expected to get created and there is a Makefile around, then "make _filename_" will create the file, by whatever _mechanism_ you want. It could do it by calling python setup.py or switch to some other "standard" mechanism in the future. 

When I type "make develop", make has a built-in ability to skip steps based on tracking dependencies. You'll notice there is also a separate target for mathics/data/op-tables.json which calls the shell script that was removed. (So now this is broken with respect to the Makefile).

Reply:
I have removed the changes to the Makefile and have restored the shell script.

Reply:
Many thanks. And also thanks for improving the setup here. 

I am good with this. @mmatera  - you too?